src/main/java/Duke/Storage/Storage.java

```diff
@@ -0,0 +1,47 @@
+package Duke.Storage;
+
+import Duke.Task.Task;
+
+import java.io.*;
```

Comment:
Avoid wildcard imports. It might be better to explicitly import the packages you're using.